_includes/appointments.html

```diff
@@ -48,6 +48,55 @@ <h4 class="subheading">{{ event.title }}</h4>
           </li>
     {% endfor %}	
           
+    {% if site.data.sitetext[page.locale].appointments.end %}	
```

Comment:
Following is currently duplicated, since I don't know how to build kind of component in Jekyll :shrug: